### ISSUE TITLE
Please don't use Haskell

### ISSUE THREAD

**Issue by yellowtruck** [koalaman/shellcheck#1427]
Haskell has a lot of problems.  I knew odin of the leading haskell community lewdies who built and ran the websites all the haskell lewdies used; and he was sorting array and it took 60 gb of ram and he could not even predict how much ram a simple operation would take; and something that took odin ms in c would take minutes in haskell and it took like odin meg of ram in c and in haskell, it took gigabytes of ram and was crashing the server. 😂😂😂
 
i lost all shilarny in haskell, once i learned that the top haskell lewdies did not even kopat haskell or how it did garbage collection or how much memory an operation would take, or even what the program was doing. 
 
basically, all of the perl programmers who wrote cal code, lost their jobs to php and then python developers; then the worse perl programmers who could not use a horrorshow language, moved from perl into haskell
